test/helpers/ProposalTest.sol

```diff
@@ -0,0 +1,121 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.18;
+
+import {
+  ICompoundTimelock
+} from "@openzeppelin/contracts/governance/extensions/GovernorTimelockCompound.sol";
+import {IERC20} from "@openzeppelin/contracts/interfaces/IERC20.sol";
+
+import {Propose} from "script/Propose.s.sol";
+import {IGovernorAlpha} from "src/interfaces/IGovernorAlpha.sol";
+import {PooltogetherGovernorTest} from "test/helpers/PooltogetherGovernorTest.sol";
+
+abstract contract AlphaProposalTest is PooltogetherGovernorTest {
+  //----------------- State and Setup ----------- //
+
+  IGovernorAlpha governorAlpha = IGovernorAlpha(GOVERNOR_ALPHA);
+  IERC20 usdcToken = IERC20(USDC_ADDRESS);
+  IERC20 radToken = IERC20(RAD_ADDRESS);
+  ICompoundTimelock timelock = ICompoundTimelock(payable(TIMELOCK));
+  uint256 initialProposalCount;
+  uint256 upgradeProposalId;
+
+  // As defined in the GovernorAlpha ProposalState Enum
+  uint8 constant PENDING = 0;
+  uint8 constant ACTIVE = 1;
+  uint8 constant DEFEATED = 3;
+  uint8 constant SUCCEEDED = 4;
+  uint8 constant QUEUED = 5;
+  uint8 constant EXECUTED = 7;
+
+  function setUp() public virtual override {
+    PooltogetherGovernorTest.setUp();
+
+    initialProposalCount = governorAlpha.proposalCount();
+
+    Propose _proposeScript = new Propose();
+    upgradeProposalId = _proposeScript.run(governorBravo);
+  }
+
+  //--------------- HELPERS ---------------//
+
+  function _assumeReceiver(address _receiver) internal {
+    assumePayable(_receiver);
+    vm.assume(
+      // We don't want the receiver to be the Timelock, as that would make our
+      // assertions less meaningful -- most of our tests want to confirm that
+      // proposals can cause tokens to be sent *from* the timelock to somewhere
+      // else.
+      _receiver != TIMELOCK
+      // We also can't have the receiver be the zero address because GTC
```

Comment:
We should see if this is the case for POOL and then update the comment or remove the `assume` statement

Reply:
https://github.com/ScopeLift/pooltogether-governor-upgrade/pull/11/commits/81f944ecd72ffe679caa0fcfc4f3e2d91e90bd7e

Reply:
I think we linked to the Governor contract when we meant to link to the POOL contract?